### ISSUE TITLE
release-please-action uses deprecated `actions/core` version, yielding warnings with every build

### ISSUE THREAD

**Issue by easimon** [google-github-actions/release-please-action#641]
### TL;DR

Github Actions deprecated the `set-output` command to set action outputs. Every action that depends on the node-js library `actions/core` uses this command under the hood to create outputs, and release-please-action is one of them. Every execution now looks similar to this:

<img width="724" alt="image" src="https://user-images.githubusercontent.com/1749657/196181208-8ba4adf7-df76-4e43-8420-adc35dd46c8a.png">

### Expected behavior

No warnings when running release-please-action.

### Observed behavior

Every execution of release-please-action emits warnings like the ones above. The number of warnings is variable (as is the number of outputs created by release-please).

The action.yaml below emits one warning.

### Action YAML

```yaml
name: Release Please
on:
  push:

jobs:
  release-please:
    runs-on: ubuntu-22.04
    outputs:
      release_created: ${{ steps.release.outputs.release_created }}
    steps:
      - uses: GoogleCloudPlatform/release-please-action@v3.5.0
        id: release
        with:
          package-name: release-please-test
          release-type: simple

  release-please-output:
    runs-on: ubuntu-22.04
    needs:
      - release-please
    steps:
      - run: echo ${{ needs.release-please.outputs.release_created }}
```

### Log output

See [example workflow run at https://github.com/easimon/bug-reproducers/actions/runs/3265332340](https://github.com/easimon/bug-reproducers/actions/runs/3265332340)

### Additional information

This action uses `actions/core@1.6.0`. To fix the issue, updating `actions/core` to at least 1.10.0 is required, which has been released only 18 days ago. 
See [Release Notes](https://github.com/actions/toolkit/blob/main/packages/core/RELEASES.md)

I don't know if there are any incompatible changes between these two versions.

**Comment by easimon** [comment]
has been fixed with #638
